config/config-example.js

```diff
@@ -368,6 +368,11 @@ exports.logchallenges = false;
  */
 exports.loguserstats = 1000 * 60 * 10; // 10 minutes
 
+/**
+ * persistbattles - Whether or not to persist battles through server restarts.
+*/
```

Comment:
Is it possible to make some kind of linter rule that catches these asterisks not lining up? Not in this PR, of course, but just in general.